lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/editions/workflow_controller.rb

```diff
@@ -119,8 +120,17 @@ def schedule_or_publish
     render "content_block_manager/content_block/editions/workflow/schedule_publishing"
   end
 
+  REVIEW_ERROR = Data.define(:attribute, :full_message)
   def publish
+    if params[:step] == NEW_BLOCK_STEPS[:review] && params[:is_confirmed].blank?
+      raise StandardError
```

Comment:
Do we need to raise here? Could we not just have the code from line 132 and down inside the conditional, and then wrap the happy path in an else?

Reply:
true! I guess it doesn't have to be an error, have turned into an if statement